src/components/apis/history-of-api/ko/runtime/api-history.html

```diff
@@ -0,0 +1,62 @@
+<div class="flex flex-row">
+    <!-- ko if: working-->
+    <spinner class="fit"></spinner>
+    <!-- /ko -->
+
+    <!-- ko ifnot: working -->
+    <div class="flex-item flex-grow animation-fade-in">
+        <!-- ko if: api -->
+        <h1>API Changlog</h1>
```

Comment:
Please remove it from widget template. We'll make it part of the default content.